docs/contributing.md

```diff
@@ -1,52 +1,81 @@
 # 🤝 Contributing
```

Comment:
Fairly large rewrite here. Most of contributing-hugo.md is moved here.,

Reply:
Thanks so much for the rewrite! :D